tutorials/3d/3d_antialiasing.rst

```diff
@@ -129,7 +162,7 @@ which is much more effective against specular aliasing. Nonetheless, exposing FX
 as an in-game option may still be worthwhile for players with low-end GPUs.
 
 FXAA introduces a moderate amount of blur when enabled (more than TAA when
-still, but less than TAA when the camera is moving).
+still, but less than TAA when the camera is moving).w
```

Comment:
I don't think you meant to add this

Reply:
Good catch, fixed :slightly_smiling_face:w